emmet-core/emmet/core/qc_tasks.py

```diff
@@ -0,0 +1,609 @@
+    # species_hash: str = Field(
+    #     None,
+    #     description="Weisfeiler Lehman (WL) graph hash using the atom species as the graph node attribute.",
+    # )
+    # coord_hash: str = Field(
+    #     None,
+    #     description="Weisfeiler Lehman (WL) graph hash using the atom coordinates as the graph node attribute.",
+    # )
+
+    # last_updated: datetime = Field(
+    #     None,
+    #     description = "Timestamp for the most recent calculation for this QChem task document",
+    # )
+
+    final_energy: float = Field(
+        None, description="Final electronic energy for the calculation (units: Hartree)"
+    )
+    enthalpy: float = Field(
+        None, description="Total enthalpy of the molecule (units: kcal/mol)"
+    )
+    entropy: float = Field(
+        None, description="Total entropy of the molecule (units: cal/mol-K"
+    )
+
+    mulliken: List[Any] = Field(
+        None, description="Mulliken atomic partial charges and partial spins"
+    )
+    resp: List[float] = Field(
+        None,
+        description="Restrained Electrostatic Potential (RESP) atomic partial charges",
+    )
+    nbo: Dict[str, Any] = Field(
+        None, description="Natural Bonding Orbital (NBO) output"
+    )
+
+    frequencies: List[float] = Field(
+        None, description="Vibrational frequencies of the molecule (units: cm^-1)"
+    )
+
+    @classmethod
+    def from_qchem_calc_doc(self, cls, calc_doc: Calculation, **kwargs) -> "OutputDoc":
+        """
+        Create a summary of QChem calculation outputs from a QChem calculation document.
+
+        Parameters
+        ----------
+        calc_doc
+            A QChem calculation document.
+        kwargs
+            Any other additional keyword arguments
+
+        Returns
+        --------
+        OutputDoc
+            The calculation output summary
+        """
+        return cls(
+            initial_molecule=calc_doc.input.initial_molecule,
+            optimized_molecule=calc_doc.output.optimized_molecule,
+            # species_hash = self.species_hash, #The three entries post this needs to be checked again
+            # coord_hash = self.coord_hash,
+            # last_updated = self.last_updated,
+            final_energy=calc_doc.output.final_energy,
+            enthalpy=calc_doc.output.enthalpy,
+            entropy=calc_doc.output.entropy,
+            mulliken=calc_doc.output.mulliken,
+            resp=calc_doc.output.resp,
+            nbo=calc_doc.output.nbo_data,
+            frequencies=calc_doc.output.frequencies,
+        )
+
+
+class InputDoc(BaseModel):
+    molecule: Molecule = Field(
+        None,
+        title="Input Structure",
+        description="Input molecule and calc details for the QChem calculation",
+    )
+
+    prev_rem_params: Dict = Field(
+        None,
+        description="Parameters from a previous qchem calculation in the series",
+    )
+
+    lev_theory: LevelOfTheory = Field(
```

Comment:
Do not like this name. Should be level_of_theory to be consistent with elsewhere in emmet.

Reply:
Also, as mentioned elsewhere, this can be broken down more granularly (e.g. to functional, basis, solvent method)